neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/embedded/impl/EmbeddedNeo4jGraphDatabaseFactory.java

```diff
@@ -24,13 +29,16 @@
  * @author Davide D'Alto &lt;davide@hibernate.org&gt;
  */
 public class EmbeddedNeo4jGraphDatabaseFactory implements GraphDatabaseServiceFactory {
+	private static final Map<String, GraphDatabaseService> GRAPH_DATABASE_SERVICE_MAP = new ConcurrentHashMap<>();
```

Comment:
I  don't understand the purpose of this.

Can you create a separate test and add it to the PR so that I can see which use case you are trying to solve?
This is the approach we work with usually, create a test first and then try to fix the problem.

Reply:
Hi @DavideD !
Main idea is support many database instances. I reminder you that one instance of embedded neo4j uses one path ONLY. In this way, Hibernate MUST manage this specialty.

About separate test. Okey. It is good and right way.